QtCreatorPackageManager.cmake

```diff
@@ -0,0 +1,7 @@
+if (NOT QT_CREATOR_SKIP_PACKAGE_MANAGER_SETUP)
+    message(STATUS "Skipping Qt Creator automatic package manager (skipping Conan and VCPKG) - check QtCreatorPackageManager.cmake for more details")
+endif()
+
+set(QT_CREATOR_SKIP_PACKAGE_MANAGER_SETUP ON)
+set(QT_CREATOR_SKIP_CONAN_SETUP ON)
+set(QT_CREATOR_SKIP_VCPKG_SETUP ON)
```

Comment:
Maybe I read the blog-post incorrectly, but doesn't `QT_CREATOR_SKIP_PACKAGE_MANAGER_SETUP` imply that checks for conan & vcpkg are skipped?

There should also be a trailing newline here.

Reply:
It should be fine to comment out the `QT_CREATOR_SKIP_VCPKG_SETUP` option then, so people know it exists still. While adding comments in there, you could also add a link to https://www.qt.io/blog/qt-creator-cmake-package-manager-auto-setup in a comment so people know better where to configure/how to configure things

Reply:
Should I leave some skip setting off? As Nerixyz mentions conan supposedly uses Qt installed on system, whilst vcpkg does bring in a different version.

Reply:
The end result of the PR should, imo, be: Qt Creator now auto-configures Conan when loading the project, but skips vcpkg. Not 100% sure how that would look

Reply:
resolved as such